packages/server/graphql/mutations/startCheckIn.ts

```diff
@@ -38,6 +39,14 @@ export default {
     if (!isTeamMember(authToken, teamId)) {
       return standardError(new Error('Team not found'), {userId: viewerId})
     }
+    const teams = await getTeamsByIds([teamId])
+    const team = teams[0]!
+    if (!isPaid(team)) {
+      const errMsg = isLocked(team)
+        ? "Wow, you're determined to use Parabol! That's awesome! Do you want to keep sneaking over the gate, or walk through the door with our Sales team?"
+        : 'Team is not paid'
+      return standardError(new Error(errMsg), {userId: viewerId})
```

Comment:
It's not in the AC for the issue so feel free to ignore, but I wonder whether we should track when this happens? We could send an event to Segment with the `isLocked` status so the Growth team can find a list of mischievous users.

Reply:
great idea, updated it to: 'Sorry! We are unable to start your meeting because your team has an overdue payment'